WordPress/src/main/java/org/wordpress/android/ui/sitecreation/verticals/NewSiteCreationVerticalsViewModel.kt

```diff
@@ -290,6 +301,9 @@ class NewSiteCreationVerticalsViewModel @Inject constructor(
         data class VerticalsModelUiState(val id: String, val title: String, val showDivider: Boolean) :
                 VerticalsListItemUiState()
 
+        data class VerticalsCustomModelUiState(val id: String, val title: String, @StringRes val subTitleResId: Int) :
```

Comment:
Shouldn't we also have `showDivider` for this? I _think_ this assumes that the custom version will always be the last item.

I suggest using the item decoration and not having this logic at all in the view holders: https://developer.android.com/reference/android/support/v7/widget/RecyclerView.ItemDecoration

Reply:
>Shouldn't we also have showDivider for this? I think this assumes that the custom version will always be the last item.

I think the custom item should always be the last item. But I guess we can add the logic to the VM to make it explicit.

>I suggest using the item decoration and not having this logic at all in the view holders: https://developer.android.com/reference/android/support/v7/widget/RecyclerView.ItemDecoration

That was my first direction, however I wasn't able to hide the divider for the last item.... Creating a custom Divider class and overriding the onDraw method felt like an overkill. Is there an easy solution?

Reply:
I've added the divider for the custom item in [1b9e045](https://github.com/wordpress-mobile/WordPress-Android/pull/8704/commits/1b9e0455442de90077563d08cb8c83ed0916c2bf). However, if we come up with a reasonable solution using ItemDecoration I'll be happy to update it.

Reply:
> I think the custom item should always be the last item. But I guess we can add the logic to the VM to make it explicit.

I don't think it's a good idea to assume implementation details. It defeats the purpose of getting the information from the API.

> That was my first direction, however I wasn't able to hide the divider for the last item.... Creating a custom Divider class and overriding the onDraw method felt like an overkill. Is there an easy solution?

I'll have to check it out.

Reply:
>I don't think it's a good idea to assume implementation details. It defeats the purpose of getting the information from the API.

I felt like the custom option should be used as a last resort so it made sense to always keep it at the last spot. But I agree with you;).